lib/api_browser/app/js/factories/template_for.js

```diff
@@ -80,6 +80,20 @@ app.provider('templateFor', function() {
     }
   });
 
+  this.register(function homePageResolver($family, $requestedTemplate) {
+    'ngInject';
+    if ($requestedTemplate === 'home') {
+      switch ($family) {
```

Comment:
Would we want to make this more generic? If the `struct.html` was renamed to `hash.html`, then all the views would just match their family.